tests/ui_tests/test_admin_page.py

```diff
@@ -94,21 +97,22 @@ def create_cluster_with_service(sdk_client_fs: ADCMClient) -> Tuple[Cluster, Ser
 @pytest.fixture()
 def create_cluster_with_component(
     create_cluster_with_service: Tuple[Cluster, Service], sdk_client_fs: ADCMClient
-) -> Tuple[Cluster, Service, Host, Provider]:
+) -> Tuple[Cluster, Service, Host, Provider, Component]:
     """Create cluster with component"""
 
     cluster, service = create_cluster_with_service
     provider_bundle = sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), 'provider'))
     provider = provider_bundle.provider_create('test_provider')
     host = provider.host_create('test-host')
     cluster.host_add(host)
-    cluster.hostcomponent_set((host, service.component(name=FIRST_COMPONENT_NAME)))
-    return cluster, service, host, provider
+    hostcomponent = cluster.hostcomponent_set((host, service.component(name=FIRST_COMPONENT_NAME)))
+    return cluster, service, host, provider, hostcomponent
```

Comment:
hostcomponent doesn't return component object directly, it'll be (I suppose) dict or list of dicts

Reply:
yep, actually there is no need in it, removed